specification/confidentialledger/Microsoft.CodeTransparency/main.tsp

```diff
@@ -28,10 +27,793 @@ namespace Microsoft.CodeTransparency;
+@doc("Historical query to get a list of entries of a given range")
+@route("/entries/txIds")
+op listEntryIds is Foundations.Operation<
+  TransactionsRange,
+  EntryIds,
+  ServiceTraits,
+  JsonServerErrors
+>;
+
+// -----------------------------------------------------------------
+// Service models
+// -----------------------------------------------------------------
+
+@removed(Versions.`2025-01-31-preview`)
+@doc("The Response body of the service parameters")
+model ParametersResult {
+  @doc("Service certificate used in TLS and for receipt verification")
+  serviceCertificate: string;
+
+  @doc("Service identifier")
+  serviceId: string;
+
+  @doc("Signature algorithm used in receipts")
+  signatureAlgorithm: string;
+
+  @doc("Tree algorithm used in receipts")
+  treeAlgorithm: string;
+}
+
+@removed(Versions.`2025-01-31-preview`)
+@doc("Response of entry submission containing the operationId.")
+model CreateEntryResult {
+  @doc("String representing the operation id submitted")
+  @pattern("^[0-9]+\\.[0-9]+$")
+  operationId: string;
+}
+
+@added(Versions.`2025-01-31-preview`)
+@doc("Response of entry submission if the response can be served immediatelly, see IETF SCITT SCRAPI draft")
+model CreateEntryCreated {
+  @doc("Status code")
+  @statusCode
+  statusCode: 201;
+
+  @doc("The MIME content type a Cose body is application/cose, containing a CoseSign1 signature.")
+  @header("Content-Type")
+  contentType: "application/cose";
+
+  @doc("Location of the transparent statement")
+  @header("Location")
+  location: string;
+
+  @doc("Transparent statement")
+  @bodyRoot
+  body: bytes;
+}
+
+@added(Versions.`2025-01-31-preview`)
+@doc("Response of entry submission containing the transparent statement id, see IETF SCITT SCRAPI draft")
+model CreateEntryPending {
+  @doc("Status code")
+  @statusCode
+  statusCode: 202;
+
+  @doc("Response content is CBOR")
+  @header("Content-Type")
+  contentType: "application/cbor";
+
+  @doc("Location of the transparent statement")
+  @header("Location")
+  location: string;
+
+  @doc("Response contains the identifier")
+  @bodyRoot
+  body: bytes;
+}
+
+@doc("Signed statement")
+model SignedStatement {
+  @doc("The MIME content type a Cose body is application/cose, containing a CoseSign1 signature, see IETF SCITT draft")
+  @header("Content-Type")
+  contentType: "application/cose";
+
+  @doc("CoseSign1 signature envelope")
+  @bodyRoot
+  body: bytes;
+}
+
+@doc("Transparent statement")
+model TransparentStatement {
+  @doc("The MIME content type a Cose body is application/cose, containing a CoseSign1 signature, see IETF SCITT draft")
+  @header("Content-Type")
+  contentType: "application/cose";
+
+  @doc("CoseSign1 signature envelope with the receipt embedded in the unprotected header")
+  @bodyRoot
+  body: bytes;
+}
+
+@doc("A ledger receipt, see IETF SCITT draft")
+model ReceiptEntry {
+  @doc("The MIME content type for raw receipt is application/cbor.")
```

Comment:
What are raw receipts? Is that the old receipt format?

Reply:
removed word `raw` but this refers to the older API as it contains `@removed`